pkg/istio/istio.go

```diff
@@ -0,0 +1,134 @@
+package istio
+
+import (
+	"context"
+
+	mf "github.com/manifestival/manifestival"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+)
+
+func GetServiceMeshNetworkPolicy() (mf.Manifest, error) {
+	networkPolicies := serviceMeshNetworkPolicies()
+	networkPoliciesUnstr, err := toUnstructured(networkPolicies)
+	if err != nil {
+		return mf.Manifest{}, err
+	}
+
+	m, err := mf.ManifestFrom(mf.Slice(networkPoliciesUnstr))
+	if err != nil {
+		return m, err
+	}
+	return m, nil
+}
+
+func IsEnabled(k kubernetes.Interface) (bool, error) {
```

Comment:
the name sounds very generic, but the impl. is for eventing.
should we change the name or the impl?

Reply:
I changed the function to accept the namespace as input